  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_leftup
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_leftdown
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_rightup
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_rightdown
    pid: {p: 100.0, i: 0.01, d: 10.0}
  gazebo_ros_control/pid_gains:
    base_to_leftup: {p: 1.0, i: 0.0, d: 1.0}
    base_to_leftdown: {p: 1.0, i: 0.0, d: 1.0}
    base_to_rightup: {p: 1.0, i: 0.0, d: 1.0}
    base_to_rightdown: {p: 1.0, i: 0.0, d: 1.0}